lib/api/slack.ex

```diff
@@ -27,20 +28,19 @@ defmodule Aprb.Api.Slack do
         raise("Unauthorized")
       end
 
-      subscriber = case Repo.get_by(Subscriber, user_id: params[:user_id]) do
+      subscriber = case Repo.get_by(Subscriber, channel_id: params[:channel_id]) do
```

Comment:
Different users subscribing gets you back the first subscriber. Sounds weird. I would just dump this model altogether and store the user who initiated the subscription on that model as `user_id` maybe?